public/index.html

```diff
@@ -4,11 +4,23 @@
   <link rel="stylesheet" type="text/css" href="styles/interface.css">
   <script src="/socket.io/socket.io.js"></script>
   <script src="scripts/main.js"></script>
+  <script src="http://code.jquery.com/jquery-1.11.0.min.js"></script>
 </head>
 
 <body>
-  <input id="input" autocomplete="off" /><button onClick="sendMessage()">SEND</button>
+
+  
   <ul id='messages'></ul>
 </body>
+<footer>
+  <input class="input" id="input" autocomplete="off" action="sendMessage()" /><button class="button" id="button" onClick="sendMessage()">SEND</button>
+  <script>
+    $("#input").keyup(function(event) {
```

Comment:
`keydown` seems more natural to me. For example, what behavior do we want if the Enter key is help down?